app/models/match_decisions/match_recommendation_dnd_staff.rb

```diff
@@ -92,9 +92,9 @@ def declined
         match.rejected!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.